test/unit/clients/ledger_client/ledger_pact_test_provider.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+// NPM dependencies
+const Pact = require('pact')
+
+// Custom dependencies
+const path = require('path')
+const port = parseInt(process.env.LEDGER_URL.match(/\d+(\.\d+)?$/g)[0], 10)
```

Comment:
In other pact tests we set random port. Shouldn't we be doing the same for ledger? Or is there anything special about it?

Reply:
In other clients the port is configurable. After a discussion with Steven we decided that instead of refactoring ledger's client and use random ports, we add a 'wrapper' that uses ledger client's port.